test/plots/cars-mpg.js

```diff
@@ -0,0 +1,35 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const data = await d3.csv("data/cars.csv", d3.autoType);
+
+  return Plot.plot({
+    x: {inset: 20, ticks: 10},
+    y: {grid: true},
+    color: {
+      type: "categorical",
+      legend: true // as soon as it's available
+    },
+    marks: [
+      Plot.ruleY([0], {filter: false}),
+      Plot.line(data,
+        Plot.binX({y: "mean"}, {
+          x: d => +d["year"],
+          y: "economy (mpg)",
+          stroke: "cylinders",
+          curve: "basis",
+          thresholds: 20
+        })
+      ),
+      Plot.dot(data,
+        Plot.reverse(Plot.sort("length", Plot.binY({ r: "count" }, {
```

Comment:
Will be nice to pass sort and reverse to bin directly when #334 lands.

```suggestion
        Plot.reverse(Plot.sort("length", Plot.binY({r: "count"}, {
```

Reply:
or #349 :)